### PR TITLE
Add director ramp up speed 

### PR DESCRIPTION
**Pull request by ysims**
Adds 

- Director ramping (slowing near ball, ramping up speed far away from ball)
- Comment out stand still in penalisation bc it causes a director bug (commented because we need to put it back in for the real robot)
- Adds a better right kick for webots

### CODE REVIEW

File: module/purpose/Soccer/src/Soccer.cpp [hunk 1]
```diff
@@ -87,7 +87,7 @@ namespace module::purpose {
                 log<NUClear::INFO>("Force playing started.");
                 cfg.force_playing = true;
                 emit<Task>(std::make_unique<FindPurpose>());
-                emit<Task>(std::unique_ptr<StandStill>(nullptr));
+                // emit<Task>(std::unique_ptr<StandStill>(nullptr));
```
Comment (PiiDish) [inline module/purpose/Soccer/src/Soccer.cpp:90]: Commented out?? Future development?

File: module/purpose/Soccer/src/Soccer.cpp [hunk 2]
```diff
@@ -68,14 +68,14 @@ namespace module::purpose {
...
-                emit<Task>(std::make_unique<StandStill>());
+                // emit<Task>(std::make_unique<StandStill>());
             }
         });
 
         on<Trigger<Unpenalisation>>().then([this](const Unpenalisation& self_unpenalisation) {
             // If the robot is unpenalised, stop standing still and find its purpose
             if (self_unpenalisation.context == GameEvents::Context::SELF) {
-                emit<Task>(std::unique_ptr<StandStill>(nullptr));
+                // emit<Task>(std::unique_ptr<StandStill>(nullptr));
```
Comment (PiiDish) [inline module/purpose/Soccer/src/Soccer.cpp:78]: Commented out?? Is this for future development?

**Review by PiiDish — CHANGES_REQUESTED** [review]

File: module/purpose/Soccer/src/Soccer.cpp (hunk 1 above)
Comment (ysims) [reply module/purpose/Soccer/src/Soccer.cpp:90]: see second dot point in the main PR comment

File: module/purpose/Soccer/src/Soccer.cpp (hunk 2 above)
Comment (ysims) [reply module/purpose/Soccer/src/Soccer.cpp:78]: see second dot point in the main PR comment

**Pull request merged**